src/linear_algebra/special_linear_group.lean

```diff
@@ -120,6 +120,14 @@ matrix.to_lin'_mul A B
   to_lin' (1 : special_linear_group n R) = linear_map.id :=
 matrix.to_lin'_one
 
+lemma det_ne_zero [nontrivial R] (g : special_linear_group n R) :
+  det g ≠ 0 :=
+by { rw g.det_coe_fun, norm_num }
+
+lemma row_nonzero [nontrivial R] (g : special_linear_group n R) (i : n):
+  g i ≠ 0 :=
```

Comment:
Not sure I like the `nonzero` in the name, since `ne_zero` is just as long and fits nicely in the naming convention. (Also added a space before the final colon.)
```suggestion
lemma row_ne_zero [nontrivial R] (g : special_linear_group n R) (i : n) :
  g i ≠ 0 :=
```